static-checks/rule-identifiers/test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+
+import re
+from collections import defaultdict
+
+from lib import util, results, oscap, versions
+
+
+profile_references = {
+    # srg, disa, stigid@PRODUCT or controls file
+    'stig': [
+        'https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cgeneral-purpose-os',  # noqa:E501
+        'https://public.cyber.mil/stigs/cci/',
+        'https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux',
+    ],
+    # ospp
+    'ospp': ['https://www.niap-ccevs.org/Profile/PP.cfm'],
+    # cis@PRODUCT or controls file
+    'cis': ['https://www.cisecurity.org/benchmark/red_hat_linux/'],
+    # anssi (controls file)
+    'anssi': ['https://cyber.gouv.fr/sites/default/files/document/linux_configuration-en-v2.pdf'],
+    # ism
+    'ism_o': ['https://www.cyber.gov.au/acsc/view-all-content/ism'],
+    # hipaa
+    'hipaa': ['https://www.gpo.gov/fdsys/pkg/CFR-2007-title45-vol1/pdf/CFR-2007-title45-vol1-chapA-subchapC.pdf'],  # noqa: E501
+}
+# Version specific references
+if versions.rhel.major == 8:
+    # pcidss
+    profile_references['pci-dss'] = ['https://www.pcisecuritystandards.org/documents/PCI_DSS_v3-2-1.pdf']  # noqa: E501
+elif versions.rhel.major == 9:
+    # pcidss
+    profile_references['pci-dss'] = ['https://www.pcisecuritystandards.org/documents/PCI_DSS_v4_0.pdf']  # noqa: E501
+
+
+# Parse from datastream all rules and all their references
+def get_rule_references():
```

Comment:
You already call the function on the module level and do it only once. I would get rid of it and just put the code into the test directly.